CHANGELOG.md

```diff
@@ -8,14 +8,21 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 ## Prerelease
 
+### Adds
+
+- Adds the `size` prop to the `Button` component. The accepted values are "small",
+  "medium", and "large". The default size value is "medium".
+
 ### Updates
 
 - Updates the colors for the `secondary` and `iconOnly` variants of the `Button` component.
 - Updates the `Tooltip` component to remove the dropshadow effect.
+  `Tabs` components based on the new `size` prop.
```

Comment:
Missing "- Updates..."

Reply:
This was also pointed out by @oliviawongnyc . I was trying to move two lines but just moved one instead. Good catch.